props/saber_BC_buttons.h

```diff
@@ -2117,6 +2120,17 @@ void DoSavedTwist() {
 
   RefPtr<BufferedWavPlayer> wav_player;
 
+  bool Event(enum BUTTON button, EVENT event) override {
+#ifdef BC_BUTTON_CLICKER
```

Comment:
I think you can move this #ifdef to surround the whole function, since you don't need it this function when BC_BUTTON_CLICKER is not defined.

Reply:
Indeed. Done.